applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py

```diff
@@ -136,17 +138,28 @@ def _ComputeLiftFromJumpCondition(self):
 
     def _ComputeLiftFromJumpCondition3D(self):
 
+        nodal_value_process = CPFApp.ComputeNodalValueProcess(self.wake_sub_model_part, ["VELOCITY"])
```

Comment:
```suggestion
        nodal_value_process = CPFApp.ComputeNodalValueProcess(self.fluid_model_part, ["VELOCITY"])
```
I think we need the fluid model part here

Reply:
Since the nodes in the trailing edge model part are also in the wake sub model part using the wake submodel part should be enough. Using the fluid model part in this case just brings more overhead.

However, you are right that if we want to check the velocity at the nodes somewhere else we can directly use the fluid model part and do everything at once here.

Reply:
My concern was that the nodal average then would be only performed using the nodes and element in the wake right? So I would expect the velocity average to not be 100% correct as it is not using all the neighbour elements in the nodes to compute the average

Where you getting good results using only the wake sub model part?

Reply:
hm, you are absolutely right ... I was not getting good results at all in 3d but I am afraid this is not the only problem :sweat_smile: in any case well done spotting it!